### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by dferber90**
Monday morning stuff.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,31 +99,31 @@
...
     "@commercetools/jest-enzyme-matchers": "1.1.2",
     "@commitlint/cli": "7.2.1",
     "@commitlint/config-conventional": "7.1.2",
-    "@percy/puppeteer": "1.0.0-beta.5",
-    "@storybook/addon-actions": "4.1.2",
-    "@storybook/addon-info": "4.1.2",
-    "@storybook/addon-knobs": "4.1.2",
-    "@storybook/addon-links": "4.1.2",
-    "@storybook/addon-options": "4.1.2",
+    "@percy/puppeteer": "1.0.0",
```
Comment (montezume) [inline package.json]: 🎉

File: package.json [hunk 2]
```diff
@@ -193,12 +193,12 @@
...
     "react-router-dom": "4.3.1",
-    "react-testing-library": "5.4.2",
+    "react-testing-library": "5.4.4",
     "react-value": "0.2.0",
-    "rimraf": "2.6.2",
-    "rollup": "1.0.0",
-    "rollup-plugin-babel": "4.1.0",
-    "rollup-plugin-cleanup": "3.0.0",
+    "rimraf": "2.6.3",
+    "rollup": "1.0.2",
```
Comment (montezume) [inline package.json]: 💅

File: package.json [hunk 3]
```diff
@@ -214,10 +214,10 @@
     "stylelint-config-standard": "18.2.0",
     "stylelint-order": "2.0.0",
     "url-loader": "1.1.2",
-    "webpack": "4.27.1",
-    "webpack-cli": "3.1.2",
+    "webpack": "4.28.3",
+    "webpack-cli": "3.2.1",
     "webpack-dev-server": "3.1.14",
```
Comment (montezume) [inline package.json]: when did this get updated?

Comment (emmenko) [reply package.json]: Yesterday 😜

Comment (montezume) [reply package.json]: how did the 🤖 get passed my percy block 🤔

Comment (emmenko) [reply package.json]: I merged that with my superpowers 💪

**Pull request merged**